src/sentry/conf/server.py

```diff
@@ -684,6 +684,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("replays.delete_replay", routing_key="replays.delete_replay"),
     Queue("counters-0", routing_key="counters-0"),
     Queue("triggers-0", routing_key="triggers-0"),
+    Queue("find_missing_codemappings", routing_key="find_missing_codemappings"),
```

Comment:
Great! No queue test complaints.

Are you going to rename the module name to `derive_code_mappings` in this PR or another one?